README.md

```diff
@@ -73,13 +73,13 @@ If you are building [product-is](https://github.com/wso2/product-is), the built
     **My Account**
 
     ```
-    regexp=(https://localhost:9443/myaccount/login|https://localhost:9443/myaccount/logout|https://localhost:9000/myaccount/login|https://localhost:9000/myaccount/logout|https://localhost:9000/myaccount)
+    regexp=(https://localhost:9443/console|https://localhost:9443/t/(.*)/console|https://localhost:9443/console/login|https://localhost:9443/t/(.*)/console/login|https://localhost:9001/console|https://localhost:9001/t/(.*)/console|https://localhost:9001/console/login|https://localhost:9001/t/(.*)/console/login)
```

Comment:
This has to be the regex of console, not myaccount.

Reply:
Already with eab38e16c09ec69679233f8056128f8cc1c46c20